### PR TITLE
Update chargeback rates docs

### PR DESCRIPTION
**Pull request by lpichler**
There are some fixes and updates to fit latest code.

cc @gtanzillo 

@miq-bot assign @abellotti

### CODE REVIEW

File: api/appendices/resource_attributes.adoc [hunk 1]
```diff
@@ -170,6 +171,10 @@ POST /api/rates
 | metric | string | Metrics, i.e. *derived_memory_available*, etc.
 | per_time | string | Measured per time, *hourly*, *daily*, *weekly*, *monthly* or *yearly*
 | per_unit | string | Measured per unit, i.e. *megabytes*, *gigabytes*, etc.
+| chargeback_rate | hash | `id` or `href` from `api/chargebacks` collection.
+| detail_measure | hash | `id` or `href` from `api/measure` collection. Details of related measure.
```
Comment (abellotti) [inline api/appendices/resource_attributes.adoc:175]: should be api/measures

File: api/reference/chargebacks_rates.adoc [hunk 2]
```diff
@@ -227,15 +485,23 @@ POST /api/rates/16
...
-  "href": "http://localhost:3000/api/rates/16",
-  "id": "16",
-  "enabled": true,
-  "description": "Allocated NICs",
-  "group": "net_io",
-  "rate": "0.02",
-  "per_time": "hourly",
-  "created_on": "2015-11-10T19:19:59Z",
-  "updated_on": "2015-11-10T19:23:57Z"
+    "href": "http://localhost:3090/api/rates/1800",
```
Comment (abellotti) [inline api/reference/chargebacks_rates.adoc:488]: example POST above is to /api/rates/16

**Comment by Fryguy** [comment]
Merging to get this in before markdown conversion.  @lpichler please do any changes in a follow up PR.

**Comment by lpichler** [comment]
thanks @abellotti for your good 👀 👍 

there is follow up PR : https://github.com/ManageIQ/manageiq_docs/pull/1451

@Fryguy thanks for merging before md conversion 👍

**Pull request merged**